samples/convert-upload-images-powervs/README.md

```diff
@@ -0,0 +1,66 @@
+#### Overview
+The `convert-upload-images-powervs` script helps to create RHEL/RHCOS/CENTOS images in PowerVS services. This script creates its own CentOS vm to do all the conversion tasks.
+
+#### What this script does - High Level Steps
+1. Spin up a CentOS instance in PowerVS using the default CentOS image [call Terraform code]
+2. Convert given RHEL/RHCOS images from the spun up CentOS instance [pvsadm tool]
+3. Upload and Import the RHEL/RHCOS OVA images from the CentOS instance [pvsadm tool]
+
+#### Environments supported
+1. Mac
+2. Windows-10 WSL or Cygwin
+3. Linux
+
+#### Prerequisite
+1. Access to any of the environments supported (mentioned in above section) with internet connectivity.
+2. IBM Cloud API key to login to the service instance
+3. Valid RedHat subscription (for RHEL image)
+
+#### Image upload command help
+```shell
+Automation for creating RHCOS/RHELimages to PowerVS Services. This is a wrapper for pvsadm tool.
+
+Usage:
+  ./convert-upload-images-powervs [ --rhel-url <url> | --rhcos-url <url> | --centos-url <url> ] --service-name  <service name> --region <bucket region> --cos-bucket <bucket name> --cos-resource-group <resource group> [ --cos-instance-name <cos instance name> ]
```

Comment:
centos-url needed ?

Reply:
the script can be used for converting and uploading centos qcow2 image too . I can remove it if it confuses